data/map altera.txt

```diff
@@ -4292,56 +4292,56 @@ system Zephyrius
 	description `	Considering everything, there is no question as to why this is called a paradise world, the city is highly limited in terms of living space, due to its nature as a city ship, and is most definitely shielded and capable of submerging, a defensive system used during the Drak-Alteran War, explaining the myth of the sunken city of Atlantis. It's one of the safest places in Alteran space, if you can afford the exorbitant housing prices.`
-	spaceport `The spaceport is placed around the city on its arms and appears to be well guarded, not only by the military personnel present, but also by the air support checking every ship before landing. Unlike your usual human spaceport, everything is highly organised here and almost eerily quiet, the personnel immediately take care of both yourself and your fleet.`
-	spaceport `	While the spaceport is not a place you feel comfortable staying in, the luxurious spaceport bar on the 3rd arm is a far nicer location, with a great view over the spaceport and surrounding ocean. The food and drinks hear leave nothing to be desired either.`
+	spaceport `The spaceport is placed around the city on its arms and appears to be well guarded, not only by the military personnel present, but also by the air support checking every ship before landing. Unlike your usual human spaceport, everything is highly organized here and almost eerily quiet, the personnel immediately take care of both yourself and your fleet.`
+	spaceport `	While the landing terminal is not a place you feel comfortable staying in, the luxurious spaceport bar on the 3rd arm is a far nicer location, offering great views of the spaceport and the surrounding ocean. The food and drinks here leave nothing to be desired.`
 	shipyard "Ancient Basic"
 	outfitter "Ancient Basic"
 
 planet Aurelis
 	landscape land/dmottl1
-	description `The innermost world of the Alteran Sector, Aurelis, is a mineral rich world that is stuck in an almost permanent winter. It is covered in oceans and hills, both frozen throughout most of the year. However, even with all the snow and ice, it is still a habitable world that is heavily engaged in mining operations.`
-	description `	The Alteran Military also has a strong presence on Aurelis, the local population is generally quite strong, as they are not only miners, but also live under higher gravity than most Alterans. The Verangian Guard, an elite Alteran army division is mainly enlisted and trained here.`
-	spaceport `Landing in the Aurelis spaceport is actually quite hard, the port itself isn't too big or busy, but the low temperatures and the amount of snowfall coming with them make sure that the spaceport is buried under snow for most of the time. The spaceport agency on Aurelis is unable to keep up with the snow whenever a storm is present or has come through recently. A newer section for the spaceport with floor heating has been built and tested, but no decision has been met yet, as the maintenance costs are quite high.`
```

Comment:
Low temperatures do not imply snow-fall, on the contrary - snow only happens in a certain temperature band and if it is colder than that, it won't snow anymore.
So I tried to express it in a more consistent way. 
I spotted another not-corrected mistake: at the end it should be "no decision has been made yet".